### ISSUE TITLE
https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-pytorch_model.bin

### ISSUE THREAD

**Issue by niu1970** [nlpyang/PreSumm#165]
[2020-05-16 17:18:03,584 INFO] https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-pytorch_model.bin not found in cache or force_download set to True, downloading to /tmp/tmpq67imxb7

I have downloaded this.bin file and put it in the tmp file,why doesn't it?

**Comment by SebastianVeile** [comment]
This is not an error. It is just informing you that it is downloading the model to "/tmp/tmpq67imxb7"

**Comment by niu1970** [comment]
Yes, but I would like to ask if I can download it offline. There may be some problems with my network, so I can't download it online. Thank you!

**Comment by mq003** [comment]
Yes, you can Download the model from offical BERT Repo and then us in the .from_pretrained function the Path to your Local BERT MODEL instead 'bert-base-uncased'. For more Information see https://huggingface.co/transformers/main_classes/model.html#transformers.PreTrainedModel.from_pretrained
